.github/workflows/integration.yml

```diff
@@ -22,17 +28,24 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
       - name: Compile Android tests
-        run: ./gradlew clean assembleAndroidTest --stacktrace -PallWarningsAsErrors=false
+        run: ./gradlew clean assembleAndroidTest $CI_GRADLE_ARG_PROPERTIES --stacktrace -PallWarningsAsErrors=false
 
   integration-tests:
     name: Integration Tests (Synapse)
+#    env:
+#      JAVA_HOME: $JAVA_HOME_11_X64
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
-        api-level: [21, 28, 30]
+        api-level: [21,23, 28, 30]
```

Comment:
Why adding 23 here, integration tests are still failing on all platforms so far :/. I'm afraid of using too much resources from GitHub, which could slow down the entire CI for all our projects

Reply:
Yes it was for testing purposes, or even better maybe we can just let none or one api-level until we fix the integration tests not to slow down the entire CI

Reply:
Yes, lets keep only 28 for now, so the it will look like `[28]`, thanks.